### PR TITLE
:arrow_up::one::one: pos_journal_pin

### PR DESCRIPTION
**Pull request by fedoranvar**

### CODE REVIEW

File: pos_journal_pin/README.rst [hunk 1]
```diff
@@ -34,14 +34,14 @@ Maintainers
 Further information
 ===================
 
-Demo: http://runbot.it-projects.info/demo/pos-addons/10.0
+Demo: http://runbot.it-projects.info/demo/pos-addons/11.0
```
Comment (KolushovAlexandr) [inline pos_journal_pin/README.rst:38]: not all references to the 10v are changed

File: pos_journal_pin/static/src/js/main.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ var _t = core._t;
                 return this._super(id);
             }
             var manager_group_id = this.pos.config.group_pos_manager_id[0];
-            var is_manager = _.include(this.pos.cashier.groups_id, manager_group_id);
+            var is_manager = _.include(this.pos.get_cashier().groups_id, manager_group_id);
```
Comment (KolushovAlexandr) [inline pos_journal_pin/static/src/js/main.js:36]: you can add yourself to contributors here

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
some minor changes are needed

File: pos_journal_pin/doc/index.rst [hunk 3]
```diff
@@ -13,7 +13,7 @@ Configuration
 Manager's PIN
 -------------
 
-Set a PIN for a POS manager as it's explained in `Confirm POS action by PIN <https://apps.odoo.com/apps/modules/10.0/pos_pin/>`_  module.
+Set a PIN for a POS manager as it's explained in `Confirm POS action by PIN <https://apps.odoo.com/apps/modules/11.0/pos_pin/>`_  module.
 As Administrator please follow next scenario:
 
 Journal
```
Comment (KolushovAlexandr) [inline pos_journal_pin/doc/index.rst:19]: Please also fix this article because there is no 'Payment Methods' menu In **point of sale** menu anymore

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
Configuration

**Review by KolushovAlexandr — APPROVED** [review]

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
Demo user cannot enter a POS.
The error message is below.

Sorry, you are not allowed to access this document. Only users with the following access level are currently allowed to do that:
- Administration/Settings
(Document model: ir.module.module) - (Operation: read, User: 5)

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
Cannot enter a value in the Tendered column on the payment screen if there is at least one payment method that requires manager permission. The issue happens not only for POS User, but also for POS Manager.

**Review by Ramil-Mukhametzyanov — APPROVED** [review]
it works well :+1:

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
price required

File: pos_journal_pin/static/src/js/main.js [hunk 4]
```diff
@@ -7,12 +8,21 @@ var core = require('web.core');
...
-var QWeb = core.qweb;
 var _t = core._t;
 
     models.load_fields('account.journal', ['ask_manager']);
 
     screens.PaymentScreenWidget.include({
+
+        click_numpad: function(button) {
+          	var paymentlines = this.pos.get_order().get_paymentlines();
+          	if (paymentlines.length > 0 || this.pos.cashregisters[0].journal.ask_manager === false) {
```
Comment (KolushovAlexandr) [inline pos_journal_pin/static/src/js/main.js]: if (paymentlines.length || !this.pos.cashregisters[0].journal.ask_manager) {

Comment (KolushovAlexandr) [reply pos_journal_pin/static/src/js/main.js]: also check the case if there is no any pymentmethods for POS. It should give an error some kind of 
**con not take property `journal` of undefined**

Comment (KolushovAlexandr) [reply pos_journal_pin/static/src/js/main.js]: fix indents Everywhere

File: pos_journal_pin/static/src/js/main.js [hunk 5]
```diff
@@ -22,14 +32,17 @@ var _t = core._t;
                     break;
                 }
             }
+
```
Comment (KolushovAlexandr) [inline pos_journal_pin/static/src/js/main.js]: keep history clean. here and below

File: pos_journal_pin/models/models.py [hunk 6]
```diff
@@ -1,10 +1,9 @@
-# -*- coding: utf-8 -*-
 # Copyright 2018 Kolushov Alexandr <https://it-projects.info/team/KolushovAlexandr>
```
Comment (KolushovAlexandr) [inline pos_journal_pin/models/models.py]: please fix the year for me, it should be 2019

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]

**Review by KolushovAlexandr — APPROVED** [review]

**Review by em230418 — APPROVED** [review]

**Comment by em230418** [comment]
> price required

@Ramil-Mukhametzyanov, looks like it is resolved: https://github.com/it-projects-llc/pos-addons/pull/1051/files#diff-63bd019ceace583987ef38b91db72a4f

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Pull request merged**